app/components/file-renderer/template.hbs

```diff
@@ -1,4 +1,7 @@
 {{#if download}}
-    <iframe src={{mfrUrl}} width={{width}} title={{t 'file_detail.mfr_iframe_title'}} height={{height}} scrolling="yes" marginheight="0" frameborder="0" allowfullscreen={{allowfullscreen}}>
+    {{#if isLoading}}
+        <div class='Loader ball-scale ball-dark'><div></div></div>
```

Comment:
Would be nice to use `{{loading-indicator}}`, if we can.

Reply:
lol, I forgot we had a component for that